### PR TITLE
Fix the wrong warmup input for sampling.

### PR DESCRIPTION
**Pull request by py4**
# Description

# Tests

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [ ] I have performed a self-review of my code.
- [ ] I have necessary comments in my code, particularly in hard-to-understand areas.
- [ ] I have made or will make corresponding changes to any relevant documentation.

### CODE REVIEW

File: tpu_commons/runner/jax/tpu_jax_runner.py [hunk 1]
```diff
@@ -327,7 +327,7 @@ def _precompile_compute_logits(self) -> None:
 
     def _precompile_sampling(self) -> None:
         logger.info("Compiling sampling with different input shapes.")
-        hsize = self.model_config.get_hidden_size()
+        hsize = self.model_config.get_vocab_size()
```
Comment (lsy323) [inline tpu_commons/runner/jax/tpu_jax_runner.py:330]: @py4 minor comment, maybe we should rename the var
